gnomad/variant_qc/pipeline.py

```diff
@@ -320,3 +328,97 @@ def generate_sib_stats(
     ).rows()
 
     return sib_stats_ht
+
+
+def train_rf_model(
+    ht: hl.Table,
+    rf_features: List[str],
+    tp_expr: hl.expr.BooleanExpression,
+    fp_expr: hl.expr.BooleanExpression,
+    fp_to_tp: float = 1.0,
+    num_trees: int = 500,
+    max_depth: int = 5,
+    test_expr: Optional[
```

Comment:
Is this option necessary? I'd have thought that just a Boolean expr would do. Seems like users can easily do 
```
rf_test_expr = hl.literal(test_expr).any(
            lambda interval: interval.contains(ht.locus)
        )
```
if they want. If using a test chrom it might be as simple as `ht.locus.contig == test_chrom`